### DISCUSSION TITLE
Where do I put this --cuda-malloc

### DISCUSSION THREAD

**Discussion by Dartterious** [lllyasviel/stable-diffusion-webui-forge#2361] [Q&A] [answered]
Me no got smarts when it come to coding. 

But I did used to know where these --XYZ things used to go when I used to use Auto1111. Not clear where it goes for Forge. I get the message as it launches:

![image](https://github.com/user-attachments/assets/b37da7a5-e55f-4428-8f29-00f295f1c320)

**Comment by Xenodermus** [comment, marked as answer]
Edit your webui_user.bat using notepad and include cuda-malloc like this:
set COMMANDLINE_ARGS=--cuda-stream --cuda-malloc --disable-gpu-warning
Save the file and restart the engine.
(note I have two other arguments there, keep the spacing)

**Reply by Dartterious** [reply]
Thank you!
